src/runtime_src/core/pcie/driver/linux/xocl/subdev/xgq_vmr.c

```diff
@@ -1448,6 +1498,13 @@ static int vmr_memory_info_query(struct platform_device *pdev,
 	return vmr_info_query_op(pdev, buf, cnt, XGQ_CMD_LOG_MEM_STATS);
 }
 
+static int vmr_log_dump_query(struct platform_device *pdev,
```

Comment:
this is not needed. the above code was designed to avoid dup code of calling vmr_info_query_op, thus we pass in different type.